MapboxNavigation/RouteMapViewController.swift

```diff
@@ -141,6 +141,8 @@ class RouteMapViewController: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         
+        annotatesSpokenInstructions = delegate?.mapViewControllerShouldAnnotateSpokenInstructions(self) ?? false
```

Comment:
Why do we have both this delegate method and an internal `annotatesSpokenInstructions` property? The public `NavigationViewController.annotatesSpokenInstructions` property should be hooked up to one of them…

https://github.com/mapbox/mapbox-navigation-ios/blob/a205ce750f3f21a03b6604de1f0e47a65dde08d5/MapboxNavigation/NavigationViewController.swift#L298-L301